tests/test_models.py

```diff
@@ -583,7 +583,7 @@ def _check_constraints(self):
 
         Entity embeddings have to have unit L2 norm.
         """
-        entity_norms = self.instance.entity_embeddings(indices=None).norm(p=2, dim=-1)
```

Comment:
necessary? doesn't entity_embeddings just provide easy access tot he same things

Reply:
This property is only available in old-style abstract model, but not in new-style. This could be changed though.

Reply:
bbca3ee17a37f18ae5d2f798a9f8bdfa0a3e699d...7fd201a3aae01ead4c756d3a9d4b63d5d7db8125

Reply:
hmm I thought it was available in both to serve as a compatibility layer

Reply:
okay it's fine if you want to skip making an update for this

Reply:
reverted in https://github.com/pykeen/pykeen/pull/955/commits/1dedcef19e17c4aa60519ceaaeff27e11024a86e